core/src/main/java/com/rim/IceField/Charge.java

```diff
@@ -1,6 +1,10 @@
 package com.rim.IceField;
 
+import com.badlogic.gdx.graphics.Texture;
+
 public class Charge extends ItemBase {
+    Texture tex = new Texture("mini2Dx.png");
```

Comment:
we said that we would divide the GUI, don't we? Since the texture is also part of the program View seems like it should also be separated

Reply:
you could add this part in the itemBaseGui and initialize with diff texture checking the tags :D